src/plugins/intel_cpu/src/nodes/roi_pooling.cpp

```diff
@@ -826,7 +826,8 @@ std::shared_ptr<ROIPooling::ROIPoolingExecutor> ROIPooling::ROIPoolingExecutor::
 
     OV_SWITCH(intel_cpu, ROIPoolingExecutorCreation, ctx, jpp.src_prc,
               OV_CASE(Precision::FP32, float),
-              OV_CASE(Precision::BF16, bfloat16_t))
+              OV_CASE(Precision::BF16, bfloat16_t),
+              OV_CASE(Precision::FP16, float16_t))
```

Comment:
I think x86 jit executor does not support fp16 precision
I highly recommend to double check the fp16 changes are working, using SPR host or sde emulator.

Reply:
@EgorDuplensky corrected